File-Formats/PDF/Create-PDF-document-in-Blazor.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Create or Generate PDF file in Blazor | Syncfusion
+title: Create or Generate PDF File in Blazor | Syncfusion
```

Comment:
The assigned issue regarding the 'title too short' has not been resolved with these changes. Could you please modify 'File' to 'document'? This adjustment might increase the title's character count.

Reply:
Check and change in all places.

Reply:
Done